VideoApp/VideoApp/Video/Participants/RemoteParticipant.swift

```diff
@@ -74,14 +74,14 @@ extension RemoteParticipant: RemoteParticipantDelegate {
     
     func remoteParticipantSwitchedOnVideoTrack(
         participant: TwilioVideo.RemoteParticipant,
-        videoTrack: TwilioVideo.RemoteVideoTrack
+        track videoTrack: TwilioVideo.RemoteVideoTrack
```

Comment:
Is `videoTrack` needed here? The definition according to the `TVIRemoteParticipant` header file shows the definition as:

```
- (void)remoteParticipant:(nonnull TVIRemoteParticipant *)participant
     switchedOnVideoTrack:(nonnull TVIRemoteVideoTrack *)track
NS_SWIFT_NAME(remoteParticipantSwitchedOnVideoTrack(participant:track:));
```

```suggestion
        track: TwilioVideo.RemoteVideoTrack
```